Activities/Python/UiPath.Python/Impl/Engine.cs

```diff
@@ -124,7 +124,7 @@ public Task Release()
         public Task<PythonObject> LoadScript(string code, CancellationToken ct)
         {
             ct.ThrowIfCancellationRequested();
-            return Task.Run(() =>
+            return RunSTA((cancellationToken) =>
```

Comment:
No need to pass ct to the func, it can use cancellationToken from the caller.

Reply:
fixed; also combined the commits to a single one.